packages/react/src/elements/components/Toast/Toast.js

```diff
@@ -2,37 +2,48 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import cx from "classnames";
 import IconButton from "../IconButton/IconButton";
+import RichText from "../RichText";
 import "./toast.scss";
 
 export default class Toast extends Component {
   render() {
-    const toastClasses = cx("hig__toast", `hig__toast--${this.props.status}`);
+    const toastClasses = cx("hig__toast", {
+      [`hig__toast--${this.props.status}`]: this.props.status
+    });
 
     return (
       <div className={toastClasses}>
+        {this.props.image && (
+          <div className="hig__toast__image-container">{this.props.image}</div>
+        )}
         <div className="hig__toast__body">
-          <div className="hig__toast__message">{this.props.children}</div>
-          <IconButton
-            title="Dismiss"
-            icon="close-notification"
-            type="flat"
-            onClick={this.props.onDismiss}
-          />
+          <div className="hig__toast__message">
+            <RichText>{this.props.children}</RichText>
+            <div className="hig__toast__dismiss">
+              <IconButton
+                title="Dismiss"
+                icon="close-notification"
+                type="transparent"
+                onClick={this.props.onDismiss}
+              />
+            </div>
+          </div>
         </div>
       </div>
     );
   }
 }
 
-Toast.defaultProps = {
-  status: "primary"
-};
-
 Toast.propTypes = {
   /**
    * Message content for the Toast
    */
   children: PropTypes.node,
+  /**
+   * An Avatar or Thumbnail to precede the notification content.
+   * If not provided, the matching status icon will be rendered.
+   */
+  image: PropTypes.node,
```

Comment:
If I understood @IsabelaKlaus correctly in our meeting, a Toast must always have a status. My thought would be to set `status: primary` as a default prop.

Reply:
I can restore it. I thought so too initially but read something in the wiki that suggested otherwise. Glad to get it confirmed in the meeting.